website/css/style.css

```diff
@@ -0,0 +1,563 @@
+/**
+reset5 © 2011 opensource.736cs.com MIT
```

Comment:
Done. Broken into: `reset.css`; `base.css` which contains the header, content, etc.; and 'cmdlet.css', which contains styles specific to the cmdlet page such as syntax, parameters, etc.